### PR TITLE
Ch2 Consolidate getting started guides

### LINKED ISSUE #93: Ch2 consolidate getting-started guides
There's a lot of duplicated information between this guide:
https://github.com/purescript/documentation/blob/master/guides/Getting-Started.md
and the getting-started chapter:
https://book.purescript.org/chapter2.html

It would be great if we could combine both of these into chapter 2, then [purescript.org](https://www.purescript.org/) could link directly to ch2 of the book, instead of the first guide.

### PR DESCRIPTION
**Pull request by milesfrain**
Fixes #93

### CODE REVIEW

**Comment by milesfrain** [comment]
Solutions: https://github.com/milesfrain/purescript-book/commit/99213b00308f23e15c4be49620c61ebe54c1bcba

**Comment by milesfrain** [comment]
Any objections to merging this? I'd also like to make sure https://github.com/purescript/documentation/pull/314 goes in first so there's no temporary loss of content.

**Comment by hdgarrood** [comment]
I think this has a good chance of being merged, but I want to review it properly first.

File: text/chapter2.md [hunk 1]
```diff
@@ -2,303 +2,115 @@
...
+There will be many more exercises in the chapters ahead, and working through those really helps with learning the material. If you're stumped by any of the exercises, please reach out to any of the community resources listed in the [Getting Help](https://book.purescript.org/chapter1.html#getting-help) section of this book, or even file an issue in this [book's repo](https://github.com/purescript-contrib/purescript-book/issues).
 
-We will use this development setup in the following chapters to compile, debug and test our code, so you should make sure that you are comfortable with the tools and techniques involved.
+Once you solve all the exercises in a chapter, you may compare your answers against those in the [solutions branch](https://github.com/purescript-contrib/purescript-book/tree/solutions). No peaking please without putting in an honest effort to solve these yourself though. And even if you are stuck, try asking a community member for help first, as we would rather give you a small hint than spoil the exercise. If you found a more elegant solution (that still only requires knowledge of covered content), please send us a PR.
```
Comment (oldfartdeveloper) [inline text/chapter2.md:116]: Change `peaking` to `peeking`.

**Comment by milesfrain** [comment]
Incorporating these changes now that purescript/documentation#314 is merged.
Happy to hear additional feedback on this chapter.
Will work on applying plan outlined in #154 to this chapter next.

**Pull request merged**